openstack/cronus/templates/nebula/_config.yaml.tpl

```diff
@@ -145,6 +145,8 @@ nebula:
     zoneID: {{ .Values.postfixDNS.zoneID }}
     auth:
       auth_url: {{ .Values.postfixDNS.auth.auth_url }}
+  postfixMgmt:
+    url: {{ .Values.postfixMgmt.url }}
```

Comment:
Would it make sense to have it under `.Values.postfix`? I see the other properties are not, so it is fine, but IMO `.Values.postfix.management.url` make sense. Up to you 👍

Reply:
Yes, valid point!
In the future, we will remove postfixDNS from Nebula altogether. This will be a good moment to reconsider this structure.